### ISSUE TITLE
multi_data.csv中，为什么攻击类型只分了四种？

### ISSUE THREAD

**Issue by KyleLeith-007** [abhinav-bhardwaj/IoT-Network-Intrusion-Detection-System-UNSW-NB15#2]
多分类的multi_data.csv中，最后四列只表示了四种攻击类别？不是一共有九种么？

**Comment by abhinav-bhardwaj** [comment]
Yes, there are 9 types in total, those 4 types that you're seeing are after feature selection. All types are considered in the final.

**Comment by KyleLeith-007** [comment]
> Yes, there are 9 types in total, those 4 types that you're seeing are after feature selection. All types are considered in the final.

Thank you, so the multi classification results described in the readme.md is nine categories instead of four categories?
